staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -192,6 +197,10 @@ func (s *SecureServingOptions) AddFlags(fs *pflag.FlagSet) {
 		"The limit that the server gives to clients for "+
 		"the maximum number of streams in an HTTP/2 connection. "+
 		"Zero means to use golang's default.")
+
+	fs.BoolVar(&s.PermitPortSharing, "permit-port-sharing", s.PermitPortSharing,
+		"If true, SO_REUSEPORT will be used when binding the port, which allows "+
+			"more than one instance to bind on the same address and port. [default=false]")
```

Comment:
Should flag documentation mention, that this is not supported on windows?

Reply:
I almost asked for that, but I think it would imply that we support windows for this binary, which I wouldn't want to promise. I think the error is good enough for now.